src/screens/Dashboard.js

```diff
@@ -232,6 +247,11 @@ class Dashboard extends Component<
             <LText semiBold style={styles.balanceTextHeader}>
               <CurrencyUnitValue unit={fiatUnit} value={totalBalance} />
             </LText>
+            <PriceChange
+              before={startPrice}
+              after={endPrice}
+              style={{ color: "white" }}
```

Comment:
mmh, so this works? 🤔 i thought with the style order above it wouldn't ?

Reply:
It works because the `color` prop has a dual use: colorize the value according to the chosen theme, or don't colorize at all if unset. That was unobvious and it's going away.